Sources/StreamChatUI/CommonViews/Attachments/AttachmentCollectionViewCell.swift

```diff
@@ -25,7 +25,7 @@ open class _AttachmentCollectionViewCell<ExtraData: ExtraDataTypes>: _Collection
     
     public var discardAction: (() -> Void)?
     
-    public private(set) var attachmentView: AttachmentView? = nil
+    public private(set) var attachmentView: AttachmentView?
```

Comment:
I think this should be open 🤔 I can see we have `public` and `open` in some components, but in order to change some child components in specific views, without changing them everywhere with the `components` config, all views should be `open private(set)` right? @VojtaStavik I think this is written in the Audits Documents, but maybe is not clear enough.

Reply:
The idea was that `attachmentView` should be created with `func attachmentView(for attachmentType: AttachmentType) -> AttachmentView?`  because the updateContent function is using it. Therefore not open, but I can open it. Want to add something @VojtaStavik ?